README.md

```diff
@@ -29,13 +29,13 @@ The following example shows a typical topology for a KMS instance:
 
 ## Using HPCS instead of Key Protect
 
-This module supports creating key rings and keys for Key Protect or Hyper Protect Crypto Services (HPCS). By default the module creates a Key Protect instance and creates the key rings and keys in that service instance, but this can be modified to use an existing HPCS instance by providing the GUID of your HPCS instance in the `var.existing_key_protect_instance_guid` input variable, and then setting the `var.create_key_protect_instance` input variable to `false`.
+This module supports creating key rings and keys for Key Protect or Hyper Protect Crypto Services (HPCS). By default the module creates a Key Protect instance and creates the key rings and keys in that service instance, but this can be modified to use an existing HPCS instance by providing the GUID of your HPCS instance in the `var.existing_kms_instance_guid` input variable, and then setting the `var.create_key_protect_instance` input variable to `false`. For more information on provisioning an HPCS instance, please see: <https://github.com/terraform-ibm-modules/terraform-ibm-hpcs>
 
-## Multiple Key Protect instances, and potential future directions for this module
+## Using Multiple Key Protect instances
 
 The strings `cos-bucket` and `ocp-cluster` are the cluster IDs for Cloud Object Storage and for the OpenShift Container Platform.
```

Comment:
cluster IDs???

Also this is now referencing `cos-bucket` and `ocp-cluster` string that are in a different section. This needs to be cleaned up

Reply:
cleaned this up and moved it up to the example in the readme that lists out the cluster/cos key structure.